Test branch for trying to get a running database

Jean-LucMartel opened the pull request:
Test branch for trying to get the database to run.  Not fully complete but has sqlite in it.

Jean-LucMartel committed:
Running branch by fixing package-lock.json and package.json

Doing this incase of merge conflicts.

Jean-LucMartel committed:
fixing merge issues

josevaldez7 commented:
Added sqlite to the project and seems to be pretty close to being fully implemented. Code overall looks good, just need to finish implementation. Also need to fix conflicts with the package.json files before we merge.

Jean-LucMartel committed:
Test database(almost working)

Tried fixing the database but ended up getting it working on another branch by another teammate.
